src/components/Tooltip/Arrow.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC } from 'react';
+
+import { TooltipPositionType } from './Tooltip.props';
+
+type ArrowProps = {
+  arrowType: TooltipPositionType;
+};
+
+export const Arrow: FC<ArrowProps> = ({ arrowType }) => {
+  switch (arrowType) {
+    case 'bottom':
+      return (
+        <svg
+          width="12"
+          height="6"
+          viewBox="0 0 12 6"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
```

Comment:
It repeats Those lines repeat on every switch case, it could be store in object :)

Reply:
I made a change - please take a look 👀